api/services/errorHandlers/tokenNotValidError.js

```diff
@@ -0,0 +1,9 @@
+const appError = require('./appError');
+
+class tokenNotValidError extends appError {
+  constructor(message) {
+    super(message || 'Token not valid.', 401);
+  }
+}
```

Comment:
Replace this message to smth else. _Token not valid_ isn't understandable for user.

Also change all class names to PascalCase (TokenNotValidError)